test/functional/kaui/admin_allowed_users_controller_test.rb

```diff
@@ -8,9 +8,33 @@ class AdminAllowedUsersControllerTest < Kaui::FunctionalTestHelper
       assert_response :success
     end
 
+    test 'should get create local only' do
+      parameters = {
+          :allowed_user => {:kb_username => 'Albator', :description => 'My french super hero'},
+          :password => 'Albator',
+          :roles => nil,
+          :external => '1'
+      }
+      post :create, parameters
+      assert_equal 'User was successfully configured', flash[:notice]
+      assert_response 302
```

Comment:
Could you verify the redirect path? Same remark for other tests below.

In some cases, we have some [complex](https://github.com/killbill/killbill-admin-ui/blob/30c60ef14925a1d2e6c3aa63b75589f258f310d9/app/controllers/kaui/engine_controller.rb#L62-L79) redirect logic that has been broken several times in the past.